discord_bot/discord_bot/add_roles.py

```diff
@@ -53,7 +53,7 @@ async def handle_adding(client, limit, discord_ids=None, channel=None, debug_cha
     dfs[leagues[0]] = get_tourneys(get_results_for_patch(patch=patch, league=leagues[0]), limit=2000)  # champ goes up to 2k
 
     if verbose:
-        await debug_channel.send(f"Loaded champ tourney data of {len(dfs[leagues[0]])} rows")
+        await debug_channel.send(f"Loaded legend tourney data of {len(dfs[leagues[0]])} rows")
```

Comment:
This will be crucial to get right, I'm a bit afraid but looks managable from your changes.